.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Use Node.js 19
```

Comment:
This should be changed to "Use Node.js 20"

Reply:
Good catch. Fixed.